ymca/src/main/webapp/js/outputLog.js

```diff
@@ -0,0 +1,53 @@
+var outputLog = 
+{
+	
+	isHover : false,
+		
+	init : function()
+	{
+		var logElements = Array.from(document
+				.getElementsByClassName("logElement"));
+		logElements.forEach(function(log) 
+		{
+			log.addEventListener("mouseover", function(){outputLog.isHover = true});
+			log.addEventListener("mouseout", function(){outputLog.isHover = false});
+		});
+	},
+	
+	append : function(level, message) 
+	{
+		var logElements = Array.from(document
+				.getElementsByClassName("logElement"));
+		logElements.forEach(function(log) 
+		{
+			var levelClass;
+			var iconHTML = '<span class="';
```

Comment:
add "glyphicon" class to span even if no icon is actualy added as it will then be padded properly